test/integration/specs/kbaseFeatureSet_data.json

```diff
@@ -57,140 +57,42 @@
                             }
                         ]
                     },
-                    {
```

Comment:
These tests seemed to be effectively redundant with slightly different data. They may be relevant, and just generated by different apps, but as far as the viewer's concerned, they seem to be similar enough.